test/e2e/restart.go

```diff
@@ -48,9 +48,7 @@ const (
 	restartPodReadyAgainTimeout = 5 * time.Minute
 )
 
-// TODO(ihmccreery): This is skipped because it was previously in
-// REBOOT_SKIP_TESTS, dates back before version control (#10078)
-var _ = Describe("Restart [Skipped]", func() {
+var _ = Describe("Restart [Disruptive]", func() {
```

Comment:
Yes.  Soon I'll be moving all `[Disruptive]` and `[Serial]` tests to their own suite so that most things run in parallel, but for now, yes, these should be running as the standard suite, unless there's something I'm missing.  We run reboot tests in no-focus runs, as far as I can tell, these should be treated the same.